Providers/Profile.swift

```diff
@@ -527,25 +526,19 @@ open class BrowserProfile: Profile {
         return hasAccount() && !rustFxA.accountManager.accountNeedsReauth()
     }
 
-    func getAccount() -> Account.FirefoxAccount? {
-        return nil
-    }
-
     var rustFxA: RustFirefoxAccounts {
         return RustFirefoxAccounts.shared
     }
 
-    func removeAccountMetadata() {
-        self.prefs.removeObjectForKey(PrefsKeys.KeyLastRemoteTabSyncTime)
-        self.keychain.removeObject(forKey: self.name + ".account")
-    }
-
-    func removeExistingAuthenticationInfo() {
-        self.keychain.setAuthenticationInfo(nil)
```

Comment:
This function was never called, even in pre-rustFxa this was not being called. Removing as I don't see the need for it.